tomcat-main/src/main/resources/defaults/base-authority-person.xml

```diff
@@ -75,6 +75,60 @@
 		<group id="contact" userecord="contact" ui-spec-inherit="false" ui-spec-prefix="contact" exists-in-services="false" />
 	</section>
 
+	<section id="makerSuppliedInformation">
+		<repeat id="pronounGroupList/pronounGroup">
+			<field id="declinedToAnswerPronoun" datatype="boolean" />
+			<repeat id="suppliedPronouns">
+				<field id="suppliedPronoun" autocomplete="true" ui-type="enum" />
+			</repeat>
+			<field id="useRestrictionPronoun" />
+		</repeat>
+		<repeat id="genderGroupList/genderGroup">
+			<field id="declinedToAnswerGender" datatype="boolean" />
+			<repeat id="suppliedGenders">
+				<field id="suppliedGender" autocomplete="true" ui-type="enum" />
+			</repeat>
+			<field id="useRestrictionGender" />
+		</repeat>
+		<repeat id="raceGroupList/raceGroup">
+			<field id="declinedToAnswerRace" datatype="boolean" />
+			<repeat id="suppliedRaces">
+				<field id="suppliedRace" autocomplete="true" ui-type="enum" />
+			</repeat>
+			<field id="useRestrictionRace" />
+		</repeat>
+		<repeat id="ethnicityGroupList/ethnicityGroup">
+			<field id="declinedToAnswerEthnicity" datatype="boolean" />
+			<repeat id="suppliedEthnicities">
+				<field id="suppliedEthnicity" autocomplete="true" ui-type="enum" />
+			</repeat>
+			<field id="useRestrictionEthnicity" />
+		</repeat>
+		<repeat id="sexualityGroupList/sexualityGroup">
+			<field id="declinedToAnswerSexuality" datatype="boolean" />
+			<repeat id="suppliedSexualities">
+				<field id="suppliedSexuality" autocomplete="true" ui-type="enum" />
+			</repeat>
+			<field id="useRestrictionSexuality" />
+		</repeat>
+		<repeat id="birthPlaceGroupList/birthPlaceGroup">
+			<field id="declinedToAnswerBirthPlace" datatype="boolean" />
+			<field id="suppliedBirthPlace" autocomplete="true" />
+			<field id="useRestrictionBirthPlace" />
+		</repeat>
+		<repeat id="birthDateGroupList/birthDateGroup">
+			<field id="declinedToAnswerBirthDate" datatype="boolean" />
+			<field id="suppliedBirthDate" ui-type="groupfield/structureddate" />
+			<field id="useRestrictionBirthDate" />
+		</repeat>
+		<repeat id="otherGroupList/otherGroup">
+			<field id="informationAuthor" autocomplete="true" />
+			<field id="informationDate" datatype="date" />
+			<field id="informationUseRestriction" autocomplete="true" ui-type="enum" />
```

Comment:
This is free text in the UI, but it has `autocomplete="true" ui-type="enum"` here. Which is right?

Reply:
@ray-lee Looking at the specs, the restriction fields should be dynamic lists so I'll need to update the others and make sure everything on the UI is correct as well.